.github/workflows/plugin-server-ci.yml

```diff
@@ -71,7 +71,9 @@ jobs:
                   python-version: 3.8
 
             - name: Install SAML (python3-saml) dependencies
-              run: sudo apt-get install libxml2-dev libxmlsec1-dev libxmlsec1-openssl
+              run: |
```

Comment:
Unrelated?

Reply:
Plugin server tests fail without this, same cause as https://github.com/PostHog/posthog/pull/7506

Reply:
(could've done it in a separate PR, but these run only when changes are made to specific files, so was more annoying not to put them here)

Reply:
Will need to pull this out into a separate PR tomorrow, but fair!

Reply:
Created a separate PR myself: https://github.com/PostHog/posthog/pull/7574

In the future, if you're fixing stuff like this prioritize getting master green ASAP - otherwise you'll create duplicate work for others.

Reply:
Makes sense, thanks!